setup.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
What is the goal of this file ? Could we remove it ?

Reply:
Initial stab at the build we abandoned in favor of perl :). I'll remove it